### PR TITLE
python/high-scores: Minor Change to 3rd possible Solution.

### PR DESCRIPTION
**Pull request by 0thomas**
Solution also works out without the first -1. Saw it first in a solution I am mentoring.

Passing all the tests without any issues.

### CODE REVIEW

File: tracks/python/exercises/high-scores/mentoring.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ class HighScores(object):
         return self.sorted_scores[-1]
 
     def personal_top_three(self):
-        return self.sorted_scores[-1:-4:-1]
+        return self.sorted_scores[:-4:-1]
```
Comment (SleeplessByte) [inline tracks/python/exercises/high-scores/mentoring.md]: Below this solution you probably want to mention that variation include:

```python
return self.sorted_scores[-1:-4:-1]
```

Might also be worthwhile to explain _why_ it works.

Comment (SleeplessByte) [reply tracks/python/exercises/high-scores/mentoring.md]: @0thomas can you make this change?

Comment (0thomas) [reply tracks/python/exercises/high-scores/mentoring.md]: @SleeplessByte Yes, sure. Was a bit busy. Here comes the new commit. Suggest any words you would like to change in the reason. I tried to express it as clearly and as simple as possible.

Comment (SleeplessByte) [reply tracks/python/exercises/high-scores/mentoring.md]: No problem! 

I think the new wording is just fine. We can always change in a future PR.

**Comment by SleeplessByte** [comment]
Thank you @0thomas :heart: (and @cmccandless for the review)

**Pull request merged**